cfgov/data_research/mortgage_utilities/s3_utils.py

```diff
@@ -16,9 +16,11 @@
 S3_SECRET = os.getenv('AWS_S3_SECRET_ACCESS_KEY')
 BASE_BUCKET = settings.AWS_STORAGE_BUCKET_NAME
```

Comment:
`settings.AWS_STORAGE_BUCKET_NAME` doesn't exist, so `BASE_BUCKET` will be `None`, which will become part of the `PUBLIC_ACCESS_BASE` URL below.

Reply:
@willbarton  Thanks for finding the AWS settings bug.

I share your reservations about the handcrafted SQL dump, but I think it's our best current option until we switch to Postgres, which could occur before this pipeline ever runs. All better options are blocked by environmental conditions on servers.

Great suggestion on who defines the dump location. 663b03c adds dump location to the function signatures so that the pipeline gets to decide.